platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_index.adoc

```diff
@@ -18,7 +18,7 @@
 The table below provides a complete list of documented Gradle Best Practices.
 
 ****
-There are currently *27 Best Practices*.
+There are currently *0 Best Practices*.
```

Comment:
? :)

Reply:
If I increment it to 28 now, then if someone else increments it also, and I rebase on top of them, it silently succeeds and our count becomes 1 less than it should.  This way it's guaranteed to conflict if someone else adds another before I rebase this, and if no one does, the 0 stands out for correction.